File: 8kyu/regexCountLowerCase.js
/*
Your task is simply to count the total number of lowercase letters in a string.
*/

// Solution 1
function lowercaseCount(str) {
    return (str.match(/[a-z]/g) || []).length;
}

// Solution 2
function lowercaseCount(str) {
    return str.replace(/[^a-z]/g,"").length;
}

File: 8kyu/regex_count_lower_case.py
# Your task is simply to count the total number of lowercase letters in a string.

import re

def lowercase_count(str):
    return len(re.findall('[a-z]',str))

# def lowercase_count(str):
#     return len(re.sub('[^a-z]', '', str))


print(lowercase_count("abc")) # 3